catalog-info.yaml

```diff
@@ -10,6 +10,33 @@ spec:
   owner: user:default/mjakl
   lifecycle: experimental
   system: integrations
+  queries:
```

Comment:
Does it make sense to extend these three examples with a `filter` that contains a placeholder (e.g., componentName). This would avoid listing all error logs, problem events, and security vulnerabilities.